protocol/app/process/errors/errors.go

```diff
@@ -0,0 +1,49 @@
+package errors
```

Comment:
Moving errors to a separate package is just creating unnecessary changes. All of the usage is still in the process package so I don't really see a need to move it.